src/PowerShellEditorServices/Services/Workspace/Handlers/ConfigurationHandler.cs

```diff
@@ -157,6 +157,12 @@ public override async Task<Unit> Handle(DidChangeConfigurationParams request, Ca
         private void SendFeatureChangesTelemetry(LanguageServerSettingsWrapper incomingSettings)
         {
             var configChanges = new Dictionary<string, bool>();
+            if (incomingSettings == null)
+            {
+                this._logger.LogTrace("Incoming settings were null");
+                return;
+            }
+
```

Comment:
Can prevent an unnecessary allocation here:

```suggestion
            if (incomingSettings is null)
            {
                this._logger.LogTrace("Incoming settings were null");
                return;
            }

            var configChanges = new Dictionary<string, bool>();
```